modules/vouch-proxy/default.nix

```diff
@@ -0,0 +1,129 @@
+{ pkgs, lib, config, ... }:
```

Comment:
This is my personal module, feel free to suggest changes or maybe even another tool.